app/Http/Controllers/Logincontroller.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+class Logincontroller extends Controller
+{
+    public function loginform(){
+        return view('login');
+    }
+    public function login(Request $request){
+          $email=$request->email;
+          $password=$request->password;
```

Comment:
we can use validation class for this. to get only validated input by `$request->validated();`